platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/EncryptingPersistenceManager.java

```diff
@@ -94,6 +96,7 @@ public void store(String pid, Dictionary properties) throws IOException {
 
   // The 2.3.7 version of the maven-bundle-plugin fails on lambdas,
   // so we need to use Java 7 constructs
+  @SuppressWarnings("squid:S1149") /* We are API bound on Dictionary */
```

Comment:
✏️ Can you put the justification comment next to the warning being suppressed such that we know it is related and so that if another one is added, it becomes clear which one is for which?

Reply:
If you still want to leave a comment by these SuppressWarnings maybe a little comment explaining what `squid:S1149` actually suppresses could be helpful for others coming across this.